src/main/resources/META-INF/resources/primefaces/ai/speechrecognition/speechrecognition.js

```diff
@@ -0,0 +1,137 @@
+PrimeFaces.widget.SpeechRecognition = PrimeFaces.widget.BaseWidget.extend({
+	
+	langs:
```

Comment:
See my previous comment about changing to `locale` and none of this will be necessary.